nucypher/policy/conditions/context.py

```diff
@@ -15,14 +15,16 @@
 USER_ADDRESS_CONTEXT = ":userAddress"
 USER_ADDRESS_EIP712_CONTEXT = ":userAddressEIP712"
 USER_ADDRESS_EIP4361_CONTEXT = ":userAddressEIP4361"
+USER_ADDRESS_EIP4361_EXTERNAL_CONTEXT = ":userAddressExternalEIP4361"
```

Comment:
Just to clarify - Is this intended for eventual deprecation in favor of `:userAddress`?

Reply:
Assuming we end up deprecating EIP712, I think we will simply end up with 2 context variables:
- `:userAddress` used for taco-specific (non single-sign on) EIP4361
- `:userAddressExternalEIP4361` (or whatever we end up calling it, if a different name is desired) used for single sign-on with EIP4361.

For the encryptor it will be the separation between relying on TACo for EIP4361 vs relying on their app's existing EIP4361 capabilities.

The other 2 context variables (`:userAddressEIP712`, `:userAddressEIP4361`) can be deprecated; more like a remove since they were never publicly released.